samples/gym/cartPole/CartPole.py

```diff
@@ -96,8 +96,8 @@ def evaluate_solver(solver_class, iterations_list, goals, trials=10):
     iterations_list = [1, 2, 3, 5, 10, 50, 100, 500, 1000, 1500, 2000, 2500]
     goals = [10, 20, 50, 100, 150, 200, 250, 300]
 
```

Comment:
Can you explain what these `iterations_list` and `goals` are?

Reply:
The evaluation loops through the `goals` and `iterations_list` to find the minimum number of MCTS iterations required to achieve each goal. A goal represents the target number of steps the CartPole must stay balanced. By iterating over the `iterations_list`, the evaluation determines the smallest number of MCTS iterations needed for the solver to consistently keep the CartPole balanced for the specified goal steps.